src/Surfnet/MessageBirdApiClientBundle/DependencyInjection/Configuration.php

```diff
@@ -28,41 +28,42 @@ class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder;
+        $treeBuilder = new TreeBuilder('surfnet_message_bird_api_client', 'string' );
```

Comment:
Comparing your change to some similar bundle Configuration files, the second param you passed ot the Tree builder is not required as it seems.

https://github.com/OpenConext/Stepup-saml-bundle/blob/develop/src/DependencyInjection/Configuration.php

Reply:
removed send parameter